rose/server/game.py

```diff
@@ -107,13 +107,24 @@ def drive_player(self, name, info):
     def print_stats(self):
         lines = ['Stats:']
         top_scorers = sorted(six.itervalues(self.players), reverse=True)
+        line = '%2s | %15s | %5s | %9s | %9s | %9s | %10s |' % \
```

Comment:
I agree with https://github.com/RedHat-Israel/ROSE/pull/472#issuecomment-1694720012, a stats struct and yaml dump makes more sense to me too.